clients/CMakeLists.txt

```diff
@@ -1,11 +1,10 @@
 project (sfizz)
 
-if (APPLE)
-	include_directories (SYSTEM /usr/local/opt/jack/include)
-	link_directories (/usr/local/opt/jack/lib)
-endif()
+find_package(PkgConfig REQUIRED)
```

Comment:
For this one I'll check the compatibility of the JACK part with MSVC later in the WE; maybe we need to disable all this in this case...

Reply:
JACK support is not important in Windows. I disable it.
I think, it would be good to default JACK to OFF on windows.

In fact, another possibility is to check for JACK_FOUND regardless of platform, and enable depending on this condition.